### DISCUSSION TITLE
How to check if UUID v5 was generated with a specific namespace?

### DISCUSSION THREAD

**Discussion by lucgagan** [uuidjs/uuid#717] [Q&A]
### Feature description

I've generated UUID with a namespace, and later I want to confirm if that ID was generated using a specific namespace. How do I do that?

**Comment by broofa** [comment]
Take a guess at the `name` and `namespace`, generate a new UUID.  If you get the same UUID as the one you have, then you guessed right.

To my knowledge that's the only way you can verify the name + namespace for an existing UUID.  The hashes used by v3 (MD5) and v5 (SHA1) are non-reversible and the RFC has no provisions for validating names or namespaces from generated IDs.   (No checksums, no validation fields of any sort.)
